test/types/errors/assignTypes.chpl

```diff
@@ -0,0 +1,22 @@
+record R {
+  type t;
+
+  proc init(type t) {
+    this.t = t;
+  }
+
+  proc init() {
+    this.t = int;
+  }
+}
+
+var myR: R(int);
+var myR2: R(int);
+var myR3 = new R();
+
+proc =(ref rhs: R(?t), lhs: R(t)) {
+  rhs.t = lhs.t;
```

Comment:
The which-hand-side?

Reply:
It causes me no end of confusion that on QWERTY keyboards we type "L" with our right hand and "R" with our left.  :D